lib/smart_proxy_dhcp_infoblox/plugin_configuration.rb

```diff
@@ -0,0 +1,37 @@
+module Proxy::DHCP::Infoblox
+  class PluginConfiguration
+    def load_classes
+      require 'infoblox'
+      require 'dhcp_common/dhcp_common'
+      require 'smart_proxy_dhcp_infoblox/host_ipv4_address_crud'
+      require 'smart_proxy_dhcp_infoblox/fixed_address_crud'
+      require 'smart_proxy_dhcp_infoblox/grid_restart'
+      require 'smart_proxy_dhcp_infoblox/unused_ips'
+      require 'smart_proxy_dhcp_infoblox/dhcp_infoblox_main'
+    end
+
+    def load_dependency_injection_wirings(c, settings)
+
+
+      c.dependency :connection, (lambda do
+                                  ::Infoblox.wapi_version = '2.0'
+                                  ::Infoblox::Connection.new(:username => settings[:infoblox_user] ,:password => settings[:infoblox_pw],
+                                                             :host => settings[:server], :ssl_opts => {:verify => false})
```

Comment:
I guess we should expose the `:ssl_opts` via settings and discourage using `:verify` false.